pkg/apis/componentconfig/edgecore/v1alpha2/default_kubelet_configuration.go

```diff
@@ -102,4 +98,19 @@ func SetDefaultsKubeletConfiguration(obj *TailoredKubeletConfiguration) {
 	obj.SeccompDefault = utilpointer.BoolPtr(false)
 	obj.MemoryThrottlingFactor = utilpointer.Float64Ptr(configv1beta1.DefaultMemoryThrottlingFactor)
 	obj.RegisterNode = utilpointer.BoolPtr(true)
+
+	if runtime.GOOS == "windows" {
```

Comment:
Why do we need to handle these parametese differently here? It seems that Kubernetes does not differentiate these param. Moreover, if we must do this, we need using `default_others` or `default_windows` to make the distinction.